README.md

```diff
@@ -49,11 +49,18 @@ jobs:
 
 ## Tips for Gradle
 
-Use junit report inside test block of gradle config:
+As Gradle uses a different build directory than Maven by default, you might need to set the `report_paths` variable:
+
+```yaml
+    `report_paths: '**/build/test-results/test/TEST-*.xml'`
 ```
-reports {
-  junitXml.isEnabled = true
+
+You also need to enable JUnit XML reports as shown below.
+
+```groovy
+test {
+  reports {
+    junitXml.enabled( true )
```

Comment:
Is there any difference between `= true` and `(true)`? If not I prefer keeping it the old way.

Reply:
The difference is not `= true` VS `(true)`, but `junitXml.enabled( true )` (which is equivalent to `junitXml.enabled true`) VS `junitXml.isEnabled = true`. The latter doesn't work in newer Gradle versions. I suspect the way you showed it before worked in very old Gradle versions as I was able to find the method as used in my new version since Gradle 4.0. I know that the way I changed this to will work for Gradle 4.0 up to the newest version.

Reply:
I found the [docs](https://docs.gradle.org/current/javadoc/org/gradle/api/reporting/Report.html#setEnabled-boolean-) for this method. Turns out `junitXml.enabled = true` also works (due to Groovy syntax rules) and is a bit simpler, will change to that.

I think `isEnabled = true` used to work in older Groovy, maybe... but it certainly does not work anymore.